docs/docsite/rst/reference_appendices/special_variables.rst

```diff
@@ -93,25 +93,24 @@ role_path
 
 Facts
 -----
-These are variables that contain information pertinent to the current host (`inventory_hostname`), they are only available if gathered first.
+These are variables that contain information pertinent to the current host (`inventory_hostname`). They are only available if gathered first.
 
 ansible_facts
     Contains any facts gathered or cached for the `inventory_hostname`
-    Facts are normally gathered by the M(setup) module automatically in a play, but any module can return facts.
+    Facts are normally gathered by the :ref:`setup <setup_module>` module automatically in a play, but any module can return facts.
 
 ansible_local
-    Contains any 'local facts' gathred or cached for the `inventory_hostname`.
+    Contains any 'local facts' gathered or cached for the `inventory_hostname`.
     The keys available depend on the custom facts created.
-    See the M(setup) module for more details.
+    See the :ref:`setup <setup_module>` module for more details.
```

Comment:
Ha! I'm so used to reading the module documentation that I didn't even see these!